examples/README.md

```diff
@@ -4,12 +4,12 @@ This directory contains example apps for each framework we support.
 
 ## Examples Development
 
-1. Create or Update an example at [`examples/{next,vue,angular}/...`](examples)
+1. Create or Update an example at [`examples/{next,react-native,vue,angular}/...`](examples)
 
    For your `aws-exports.js`, you can [reference an existing or create a new environment](../environments/README.md).
 
-1. Run your example at monorep root: `yarn {react,vue,angular}-example dev`
+1. Run your example at monorep root: `yarn {react,react-native,vue,angular}-example dev`
 1. Visit your example (e.g. <http://localhost:3000/ui/components/authenticator/sign-up-with-username>)
```

Comment:
Wonder if we should update this line as well since it is specific to browser frameworks